app/src/main/java/co/tinode/tindroid/MessagesFragment.java

```diff
@@ -10,7 +10,7 @@
 import android.database.Cursor;
 import android.graphics.Bitmap;
 import android.graphics.BitmapFactory;
-import androidx.exifinterface.media.ExifInterface;
+import android.media.ExifInterface;
```

Comment:
From https://developer.android.com/reference/android/media/ExifInterface

<i>Note: It is recommended to use the AndroidX ExifInterface Library since it is a superset of this class. </i>

Why do you want to make this change?

Reply:
Because androidx.exifinterface.media.ExifInterface not found

Reply:
Then you are doing something wrong. Please fix your build environment.